Apps/WebClient/src/ClientApp/src/store/modules/laboratory/actions.ts

```diff
@@ -336,7 +336,7 @@ export const actions: LaboratoryActions = {
                 customBannerError.description =
                     "The information you entered does not match our records. Please try again.";
                 customBannerError.detail =
-                    "Please note that it can take up to 48 hours from the time of test before a result is available. If it has been at least 48 hours since you tested, please contact the COVID-19 Results Line (1‐833‐707‐2792) to investigate the issue.";
+                    "Please note that it can take up to 48 hours from the time of test before a result is available. If it has been at least 48 hours since you tested, please contact the COVID‑19 Results Line (1‐833‐707‐2792) to investigate the issue.";
```

Comment:
Hyphens on phone number missed here.

Reply:
Actually they were already changed to non-breaking before but the Covid-19 had been missed.